### PR TITLE
Upgrade Twitter and HN Search

### PR DESCRIPTION
**Pull request by cdmwebs**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,4 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'hacker_news_search', github: 'cdmwebs/hacker_news_search'
```
Comment (mguterl) [inline Gemfile]: I don't think this won't work for the gem release. Can we get your changes pushed upstream?

**Comment by cdmwebs** [comment]
@mguterl I did get them merged, [but](https://github.com/ryanatwork/hacker_news_search/pull/7#issuecomment-90385019).

**Comment by mguterl** [comment]
@cdmwebs that sucks :-1:

**Pull request closed without merging**